bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -354,7 +355,7 @@ JsErrorCode DefineHostCallback(
 JsErrorCode RedirectConsoleToDebugger(DebugProtocolHandler *handler)
 {
     // Get global.console object. If that is not defined just return, otherwise get the debugger.console and patch with global.console object
-
+    fprintf(stdout, "saasa");
```

Comment:
Was this added for testing? It doesn't seem like this was meant to be checked in.

Reply:
Sorry, removed it